src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrPusher.cs

```diff
@@ -163,47 +162,7 @@ private async Task<bool> CheckCommitAvailability(string gitHubApiPat, Cancellati
         return result;
     }
 
-    private CommitSearchArguments GetCommitSearchArguments(string uri, string sha)
-    {
-        var repoType = GitRepoTypeParser.ParseFromUri(uri);
-        
-        if (repoType == GitRepoType.AzureDevOps)
-        {
-            string[] repoParts = uri.Substring(uri.LastIndexOf('/')).Split('-', 2);
-
-            if (repoParts.Length != 2)
-            {
-                throw new Exception($"Invalid URI in source manifest. Repo '{uri}' does not end with the expected <GH organization>-<GH repo> format.");
-            }
-
-            string org = repoParts[0];
-            string repo = repoParts[1];
-
-            // The internal Nuget.Client repo has suffix which needs to be accounted for.
-            const string trustedSuffix = "-Trusted";
-            if (uri.EndsWith(trustedSuffix, StringComparison.OrdinalIgnoreCase))
-            {
-                repo = repo.Substring(0, repo.Length - trustedSuffix.Length);
-            }
-
-            return new CommitSearchArguments(repo, org, sha);
-        }
-        else if (repoType == GitRepoType.GitHub)
-        {
-            string[] repoParts = uri.Substring(Constants.GitHubUrlPrefix.Length).Split('/', StringSplitOptions.RemoveEmptyEntries);
-
-            if (repoParts.Length != 2)
-            {
-                throw new Exception($"Invalid URI in source manifest. Repo '{uri}' does not end with the expected <GH organization>/<GH repo> format.");
-            }
-
-            return new CommitSearchArguments(repoParts[1], repoParts[0], sha);
-        }
-
-        throw new Exception("Unsupported format of repository url " + uri);
-    }
-
-    private string GetGraphQlIdentifier(string repoName) => repoName.Replace("-", string.Empty).Replace(".", string.Empty);
+    private static string GetGraphQlIdentifier(string repoName) => repoName.Replace("-", string.Empty).Replace(".", string.Empty);
```

Comment:
I think `Replace` accepts `null` for this purpose
```suggestion
    private static string GetGraphQlIdentifier(string repoName) => repoName.Replace("-", null).Replace(".", null);
```